content/docs/tinacms-context.md

```diff
@@ -129,9 +129,72 @@ export const getStaticProps = async () => {
 - `query` and `variables`: These are the same values that you would use for the [backend data-fetching](/docs/features/data-fetching/).
 - `data`: This is the production value that gets passed through to the response unchanged in production.
 
+## The `RouteMapping` Plugin
+
+The `RouteMapping` Plugin is used in the Collection List pages to route to a Document's "Contextual Editing" view rather than the default "Fullscreen Editing" view.
+
+```ts
+RouteMappingPlugin(mapper: (collection: Collection, document: Document) => string | undefined)
+```
+
+The `RouteMappingPlugin` accepts a single argument - the `mapper` function - that is run against each `document` and `collection` in a Collection List.
+
+- If the `mapper` returns a `string`, that `string` is used as the Document's route rather than the default one.
+- If the `mapper` returns `undefined`, the route defaults to the Document's "Fullscreen Editing" page.
+
+This is an example of the `RouteMappingPlugin` added to our `tina-cloud-starter` template:
+
+```tsx
+import { RouteMappingPlugin } from 'tinacms';
```

Comment:
This should only be loaded in edit mode. It might be best to change this example to 
```
cmsCallback={cms => {
               import('tinacms').then((package)=>{
                  cms.plugins.add(package.RouteMapperPlugin(...))
                  })
            }}
 ```
 We don't want users to increase their bundle size.